### ISSUE TITLE
(User feedback) P1: VoiceOver announcements not working on iPhone, (yet works on iPad).

### ISSUE THREAD

**Issue by gbarkerz** [gbarkerz/AccessibleSolitaire#90]
From user:

"On the iPhone, VoiceOver doesn't announce the cards I'm turning over at all. Not after selecting the next card, not after revealing from the pile of dealt cards. On the iPad, everything works as before."
